modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/download/UpdateEpisodeDetailsTask.kt

```diff
@@ -54,6 +58,13 @@ class UpdateEpisodeDetailsTask @AssistedInject constructor(
                 .build()
             WorkManager.getInstance(context).beginUniqueWork(TASK_NAME, ExistingWorkPolicy.APPEND, workRequest).enqueue()
         }
+
+        /**
+         * Don't bother calling episodes that are downloaded as the download task checks the content type.
+         */
```

Comment:
It feels like this is more of a comment (`//`) than KDoc (`/** */`).

Reply:
Good point. I have made that change. Thanks for the feedback.